RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -61,27 +61,97 @@ public bool Generate(ProcessedArgs args)
                 throw new ArgumentNullException("args");
             }
 
-            IPackage package = this.packageHandler.FetchPackage(args.PackageId, args.PackageVersion);
-            if (package == null)
+            IPackage targetPackage = this.packageHandler.FetchPackage(args.PackageId, args.PackageVersion);
+            IEnumerable<IPackage> dependencyPackages = this.packageHandler.GetInstalledDependencies(targetPackage);
```

Comment:
There's no point in trying to fetch the dependency packages if _targetPackage_ is null.